test/end-to-end/stock/stock.aggregate_consumption.page.js

```diff
@@ -0,0 +1,164 @@
+/* global element, by */
+const FU = require('../shared/FormUtils');
+const GU = require('../shared/GridUtils');
+const components = require('../shared/components');
+const SharedStockPage = require('./stock.shared.page');
+
+function StockAggregateConsumptionPage() {
+  const page = this;
+  const gridId = 'aggregated-consumption-grid';
+  const modalGridId = 'lots-grid';
+
+  // change current depot
+  page.changeDepot = SharedStockPage.setDepot;
+
+  // the grid id
+  page.gridId = gridId;
+  page.modalGridId = modalGridId;
+
+  // create modal
+  page.showCreateModal = async isNewRequisition => {
```

Comment:
Do we need the `isNewRequisition` flag here?

I think you may have copied this from somewhere.  Can you remove the methods you don't actually use from this page?